cmd/dqa/results/file.go

```diff
@@ -230,6 +258,43 @@ type Result struct {
 	fileVersion uint8
 }
 
+func (r *Result) Migrate() *Result {
+
+	res := &Result{
+		Model:        r.Model,
+		ModelVersion: r.ModelVersion,
+		DataVersion:  r.DataVersion,
+		DQAVersion:   r.DQAVersion,
+		Table:        r.Table,
+		Field:        r.Field,
+		CheckCode:    r.CheckCode,
+		CheckType:    r.CheckType,
+		Finding:      r.Finding,
+		Prevalence:   r.Prevalence,
+		Rank:         r.Rank,
+		rank:         r.rank,
+		Cause:        r.Cause,
+		Status:       r.Status,
+
+		fileVersion: currentFileVersion,
+	}
+
+	// Added in later version.
+	if r.fileVersion < FileVersion2 {
+		res.GithubID = "N/A"
+	}
+
+	if r.fileVersion < FileVersion3 {
+		res.Method = "N/A"
+	}
+
+	if r.fileVersion < FileVersion4 {
+		res.CheckAlias = "N/A"
+	}
```

Comment:
Okay yea, I wasn't sure if having "N/A" set as an explicit value (or some other default value) would be good to distinguish between a migrated field and a field that just maybe was blank to begin with? But I guess all the fields should technically be filled in, so that makes sense